### PR TITLE
[instrument] Optionally log build info to stdout

### PR DESCRIPTION
**Pull request by jeromefroe**
This commit updates the `instrument` package to be able to log the build info to stdout instead of stderr. GKE logs written to stdout are classified as `INFO` logs in Google Cloud Logging whereas logs written to `stderr` are classified as `ERROR` logs so build info logs are currently classified as `ERROR` logs which doesn't feel appropriate.

### CODE REVIEW

File: src/x/instrument/build.go [hunk 1]
```diff
@@ -52,6 +53,10 @@ var (
 	// set to a non-empty string, we log the build information at process startup.
 	LogBuildInfoAtStartup string
 
+	// LogBuildInfoToStdout controls whether we log build information to stdout or stderr.
+	// If it and LogBuildInfoAtStartup are set to a non-empty string
+	LogBuildInfoToStdout string
```
Comment (ryanhall07) [inline src/x/instrument/build.go:58]: lol `LogBuildInfoToStdout=false` -> true

**Review by ryanhall07 — APPROVED** [review]
thanks!

File: src/x/instrument/build.go [hunk 2]
```diff
@@ -52,6 +53,10 @@ var (
 	// set to a non-empty string, we log the build information at process startup.
 	LogBuildInfoAtStartup string
 
+	// LogBuildInfoToStdout controls whether we log build information to stdout or stderr.
+	// If it and LogBuildInfoAtStartup are set to a non-empty string
```
Comment (jjshanks) [inline src/x/instrument/build.go:57]: Seems to be missing a word?

**Review by jjshanks — APPROVED** [review]

File: src/x/instrument/build.go (hunk 1 above)
Comment (jeromefroe) [reply src/x/instrument/build.go:60]: Yea, this is admittedly kinda weird, but I wanted to keep it consistent with `LogBuildInfoAtStartup`

**Pull request merged**